package.json

```diff
@@ -8,16 +8,16 @@
   },
   "bugs": "https://github.com/TryGhost/Ghost/issues",
   "private": true,
-  "version": "0.11.3",
+  "version": "0.11.4",
```

Comment:
Updated to 0.11.7 and still runs fine 😉

Maybe @cobyism what's to get this in.